battery.sh

```diff
@@ -568,3 +571,36 @@ if [[ "$action" == "logs" ]]; then
 	exit 0
 
 fi
+
+# Battery calibration
+if [[ "$action" == "calibrate" ]]; then
+	log "Starting calibration"
+
+	# Stop the maintaining
+	battery maintain stop
+
+	# Discharge battery to 15%
+	battery discharge 15
+
+	while true; do
```

Comment:
For this, please use the built in `battery charge` command. When `battery charge 100` is done, the command exits and lets your script continue

Reply:
If I use "battery charge 100" won't it start battery discharging after the execution? I need to wait 1 hour before discharging process.